src/openrct2/scripting/bindings/game/ScContext.hpp

```diff
@@ -301,6 +301,16 @@ namespace OpenRCT2::Scripting
             return 1;
         }
 
+#    ifdef _MSC_VER
+        __declspec(noinline)
+#    endif
+            std::shared_ptr<ScDisposable> CreateSubscription(HOOK_TYPE hookType, const DukValue& callback)
```

Comment:
Could you add a comment explaining to why this evil is necessary, someone might see this and think to undo it.